service/src/test/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateSentinelAlertRulesStepTest.java

```diff
@@ -0,0 +1,113 @@
+import bio.terra.profile.model.ProfileModel;
+import bio.terra.stairway.StepResult;
+import com.azure.resourcemanager.securityinsights.fluent.models.AlertRuleInner;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@Tag("unit")
+class CreateSentinelAlertRulesStepTest extends BaseStepTest {
+
+  @Mock LandingZoneProtectedDataConfiguration mockLandingZoneProtectedDataConfiguration;
+  @Mock AlertRulesHelper mockAlertRuleAdapter;
+
+  @Test
+  void doStepSuccess() throws InterruptedException {
+    setupFlightContext(
+        mockFlightContext,
+        Map.of(
+            LandingZoneFlightMapKeys.BILLING_PROFILE,
+            new ProfileModel().id(UUID.randomUUID()),
+            LandingZoneFlightMapKeys.LANDING_ZONE_ID,
+            LANDING_ZONE_ID),
+        Map.of(
+            GetManagedResourceGroupInfo.TARGET_MRG_KEY,
+            ResourceStepFixture.createDefaultMrg(),
+            CreateLogAnalyticsWorkspaceStep.LOG_ANALYTICS_RESOURCE_KEY,
+            buildLandingZoneResource()));
+    var scheduledRuleIds = List.of(UUID.randomUUID().toString(), UUID.randomUUID().toString());
+    var mlRuleIds = List.of(UUID.randomUUID().toString(), UUID.randomUUID().toString());
+    var nrtRuleIds = List.of(UUID.randomUUID().toString(), UUID.randomUUID().toString());
+    when(mockLandingZoneProtectedDataConfiguration.getSentinelScheduledAlertRuleTemplateIds())
+        .thenReturn(scheduledRuleIds);
+    when(mockLandingZoneProtectedDataConfiguration.getSentinelMlRuleTemplateIds())
+        .thenReturn(mlRuleIds);
+    when(mockLandingZoneProtectedDataConfiguration.getSentinelNrtRuleTemplateIds())
+        .thenReturn(nrtRuleIds);
+    var mockRule = mock(AlertRuleInner.class);
+    when(mockAlertRuleAdapter.buildScheduledAlertRuleFromTemplate(
+            anyString(), anyString(), anyString()))
+        .thenReturn(mockRule);
+
+    var createSentinelAlertRulesStep =
+        new CreateSentinelAlertRulesStep(
+            mockArmManagers,
+            mockParametersResolver,
+            mockResourceNameGenerator,
+            mockAlertRuleAdapter,
+            mockLandingZoneProtectedDataConfiguration);
+
+    var result = createSentinelAlertRulesStep.doStep(mockFlightContext);
+
+    assertThat(result, equalTo(StepResult.getStepResultSuccess()));
+    scheduledRuleIds.forEach(
+        u ->
+            verify(mockAlertRuleAdapter, times(1))
+                .createAlertRule(any(), eq(u), anyString(), anyString()));
+    nrtRuleIds.forEach(
+        u ->
+            verify(mockAlertRuleAdapter, times(1))
+                .createAlertRule(any(), eq(u), anyString(), anyString()));
+    mlRuleIds.forEach(
+        u ->
+            verify(mockAlertRuleAdapter, times(1))
```

Comment:
Nit/fyi: `times(1)` is the default behavior for `verify` - so you an just omit the argument, and get the same result